NetKAN/CustomAsteroids.netkan

```diff
@@ -23,17 +26,22 @@
 			"filter": "config"
 		}
 	],
-	"name": "Custom Asteroids",
-	"abstract": "Custom Asteroids is a mod for KSP that expands the stock asteroid functionality. Under Custom Asteroids most asteroids will not be on a flyby course for Kerbin. Instead, the mod places asteroids in specific populations throughout the stock kerbol system.",
-    "x_netkan_override" : [
-        {
-            "version" : "v1.1.0",
-            "override" : {
-                "ksp_version_min": "0.25",
-                "ksp_version_max": "1.0.4"
-            },
-            "delete" : [ "ksp_version" ]
-        }
-    ],
+	"x_netkan_override" : [
+		{
+			"version" : "v1.1.0",
+			"override" : {
+				"ksp_version_min": "0.25",
+				"ksp_version_max": "1.0.4"
+			},
+			"delete" : [ "ksp_version" ]
+		},
+		{
+			"version" : "v1.2.0",
+			"override" : {
+				"ksp_version_min": "1.0.0"
```

Comment:
Are you *sure* it's not "hardcoding a ksp_version_min, not hardcoding ksp_version_max"? I'm pretty sure anything not mentioned in an override statement (e.g., using "delete") gets inherited from the main body.